compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/CanvasLayersComposeScene.skiko.kt

```diff
@@ -241,6 +241,13 @@ private class CanvasLayersComposeSceneImpl(
         return result
     }
 
+    override fun cancelPointerInput() {
+        mainOwner.onCancelPointerInput()
```

Comment:
```suggestion
        gestureOwner?.onCancelPointerInput()
```

It doesn't look correct when we cancel pointers in the main owner, which can be under current gesture owner (Popup/Dialog)

@MatkovIvan will correct me if I am wrong.

Reply:
Yes. Calling just `mainOwner` is not correct here (it's a good reason to add unit test for that).

`gestureOwner` should be ok🤔 Probably. I'd suggest to call it for all owners to be on the safe side

Reply:
Updated. Test added